modules/caddyhttp/fileserver/staticfiles.go

```diff
@@ -79,6 +80,14 @@ type FileServer struct {
 	// a 404 error. By default, this is false (disabled).
 	PassThru bool `json:"pass_thru,omitempty"`
 
+	// Selection of encoders to use to check for precompressed files.
+	PrecompressRaw caddy.ModuleMap `json:"encodings,omitempty" caddy:"namespace=http.precompressed"`
```

Comment:
I usually prefer that the json name matches the field name (except "Raw" suffixes). So like `json:"precompressed,omitempty" and a field name of `PrecompressedRaw` or something like that.

Reply:
Again a thing I forgot to change along the namespace >.>